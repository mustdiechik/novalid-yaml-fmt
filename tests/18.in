data:
  {{- if .Values.ingress.hostname }}
  - host: {{ .Values.ingress.hostname }}
    foo
    bar
  - zzz:
    xxx:
    yyy:
  {{ if 1 }}
  - xxx:
    for
    bar


data2:
  {{ printf }}
    - host: {{ .Values.ingress.hostname }}
      foo: 1
      bar: 2
  {{ if 2 }}
    - yyy: 3
      for: 4
      bar: 5
