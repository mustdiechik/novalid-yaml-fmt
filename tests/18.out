data:
  {{- if .Values.ingress.hostname }}
    - host: {{ .Values.ingress.hostname }}
      foo
      bar
    - xxx:
      for
      bar


data:
  {{- if .Values.ingress.hostname }}
    - host: {{ .Values.ingress.hostname }}
      foo
      bar
    - yyy:
      for
      bar
